lib/pbench/test/unit/server/query_apis/test_datasets_mappings.py

```diff
@@ -127,4 +127,4 @@ def test_with_db_error(self, client, server_config, database_error):
         with client:
             response = client.get(f"{server_config.rest_uri}/datasets/mappings/summary")
             assert response.status_code == HTTPStatus.INTERNAL_SERVER_ERROR
-            assert response.json["message"] == "Internal Server Error"
+            assert response.json["message"] == "INTERNAL ERROR"
```

Comment:
Should we be comparing against `HTTPStatus.INTERNAL_SERVER_ERROR.phrase`, like we do in the other test modules?